<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0" colspan="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="title">
        <property name="text">
         <string>Find Student</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="periodComboBox">
        <property name="currentText">
         <string>A (Day 1 Period 1)</string>
        </property>
        <item>
         <property name="text">
          <string>A (Day 1 Period 1)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B (Day 1 Period 2)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C  (Day 1 Period 3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D (Day 1 Period 4)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E (Day 2 Period 1)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F (Day 2 Period 2)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G (Day 2 Period 3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H (Day 2 Period 4)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="picture">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="search"/>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QTableWidget" name="studentTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="importStudentsButton">
          <property name="text">
           <string>Import Students</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="addStudentButton">
          <property name="text">
           <string>Add Student</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
